### PR TITLE
doc: azure w/ subscription and coreos option as env

### PR DESCRIPTION
**Pull request by eljefedelrodeodeljefe**
Enhances handling the Azure getting started guide for bigger Azure clients. This now can handle mutliple subscriptions by passing an optional environment variable.

This works for creating and deletion of a cluster.

### CODE REVIEW

**Comment by smarterclayton** [comment]
@errordeveloper can you review this?

File: docs/getting-started-guides/coreos/azure/README.md [hunk 1]
```diff
@@ -78,6 +78,13 @@ Now, all you need to do is:
 ```
 
 This script will provision a cluster suitable for production use, where there is a ring of 3 dedicated etcd nodes: 1 kubernetes master and 2 kubernetes nodes. The `kube-00` VM will be the master, your work loads are only to be deployed on the nodes, `kube-01` and `kube-02`. Initially, all VMs are single-core, to ensure a user of the free tier can reproduce it without paying extra. I will show how to add more bigger VMs later.
+If you need to pass Azure specific options for the creation script you can do this via additional environment variables e.g.
+
+```
+AZ_SUBSCRIPTION=<id> AZ_LOCATION=East US ./create-kubernetes-cluster.js
```
Comment (errordeveloper) [inline docs/getting-started-guides/coreos/azure/README.md]: I believe you must quote the location name

**Comment by errordeveloper** [comment]
See comment about `AZ_LOCATION`, the rest LGTM.

File: docs/getting-started-guides/coreos/azure/README.md (hunk 1 above)
Comment (eljefedelrodeodeljefe) [inline docs/getting-started-guides/coreos/azure/README.md]: That's right. Will update.

**Comment by eljefedelrodeodeljefe** [comment]
Done. Do you guys prefer squashed commits?

**Comment by errordeveloper** [comment]
Yes, squash it please.

**Comment by eljefedelrodeodeljefe** [comment]
@errordeveloper done.

**Comment by errordeveloper** [comment]
@eljefedelrodeodeljefe thanks! @smarterclayton can merge.

**Pull request merged**